.github/workflows/ci.yml

```diff
@@ -28,6 +28,6 @@ jobs:
 
       - run: bundle exec rake test
         env:
-          STREAM_CHAT_API_KEY: ${{ secrets.STREAM_CHAT_API_KEY }}
-          STREAM_CHAT_API_SECRET: ${{ secrets.STREAM_CHAT_API_SECRET }}
+          STREAM_KEY: ${{ secrets.STREAM_CHAT_API_KEY }}
+          STREAM_SECRET: ${{ secrets.STREAM_CHAT_API_SECRET }}
```

Comment:
Can you please also check `STREAM_CHAT_URL` vs `STREAM_CHAT_API_HOST` usage in code?

Reply:
@ffenix113
![image](https://user-images.githubusercontent.com/19969687/150948283-adae0ce1-e9e4-4cd2-88d5-fdf5a1856b14.png)

Reply:
Thanks, but `STREAM_CHAT_API_HOST` is used in CI configuration and tests, while `STREAM_CHAT_URL` is only used when creating client. Could you please check which one should be used everywhere?

Reply:
Sure. I attached a picture of the ones that should be used. I removed `STREAM_CHAT_API_HOST` from the CI, it doesn't make sense for us. I've checked the usages.

![image](https://user-images.githubusercontent.com/19969687/150956538-1abb6573-66cb-46db-8aa0-381ad8f6459f.png)

Reply:
Thanks. Added two comments(maybe forgot to push).

When they are done - let's merge it.